distro/sql-script/pom.xml

```diff
@@ -118,8 +129,16 @@
                   <overWrite>true</overWrite>
                   <outputDirectory>target/camunda-engine-${project.version}</outputDirectory>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.camunda.bpm.distro</groupId>
+                  <artifactId>camunda-sql-scripts</artifactId>
+                  <version>7.15.0</version>
```

Comment:
Should we include an explanation why 7.15.0 is used similar to the one above to discourage changing it?

```suggestion
                  <!-- this is the baseline for the Liquibase changelog, do not change -->
                  <version>7.15.0</version>
```